Assignment3/src/Employee.java

```diff
@@ -0,0 +1,41 @@
+public class Employee {
+
+    String Name;
```

Comment:
The properties are typically marked private and exposed thru getters and setters.